packages/default-evaluate-options/test/test.js

```diff
@@ -5,7 +5,7 @@ import makeDefaultEvaluateOptions from '../src';
 test('sanity', async t => {
   try {
     const options = makeDefaultEvaluateOptions();
-    t.assert(Array.isArray(options.shims), `options.shims is an array`);
+    t.assert(Array.isArray(options.shims), 'options.shims is an array');
```

Comment:
I'm not sure whether you are making these changes because of a lint rule?  If so, I thought we had turned that one off. It's one of the objectionable ones from AirBNB.  If you actaully generally prefer that (but don't require that) then that's fine :)

Reply:
Not a lint rule and not a general preference. I prefer it here because we'll normally write a `details` tagged template literal here. I didn't want to make it easy to insert a substitution value and miss the absence of the `details` tag. That would leak info in a way no one would catch until too late.